ui-v2/package.json

```diff
@@ -22,9 +22,12 @@
     "start:consul": "ember serve --proxy=${CONSUL_HTTP_ADDR:-http://localhost:8500} --port=${EMBER_SERVE_PORT:-4200} --live-reload-port=${EMBER_LIVE_RELOAD_PORT:-7020}",
     "start:api": "api-double --dir ./node_modules/@hashicorp/consul-api-double",
     "test": "ember test --test-port=${EMBER_TEST_PORT:-7357}",
+    "test:oss": "CONSUL_NSPACES_ENABLED=0 ember test --test-port=${EMBER_TEST_PORT:-7357}",
```

Comment:
Does one of these scripts need to have the `CONSUL_NSPACES_ENABLED` set to true?

Reply:
A good question! I should have explained this also.

I tried to touch as little as possible, so the old `yarn run test` script is still uses `CONSUL_NSPACES_ENABLED=true` by default, so basically if you don't set it on the CLI it defaults to `true`.

I'd actually like to chat with you about this and see what you think. I think we should decide which run (oss or ent) we want as default when you run `make test` - I'm almost on the side of it being the ent one, but let's chat about it when we next hook up. Whichever we decide I think if we change it we should do it in another PR.